content/pt-br/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,182 @@
+---
+title: EndpointSlices
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
+
+_EndpointSlices_ entregam uma maneira simples de rastrear endpoints de rede através de um cluster Kubernetes. Eles oferecem uma alternativa mais escalável e extensa aos Endpoints.
+<!-- body -->
+
+## Motivação
+
+A Endpoints API oferece uma maneira simples e direta de rastremento de endpoints de rede no Kubernetes. Infelizmente à medida que os clusters Kubernetes e os {{< glossary_tooltip text="Serviços" term_id="service" >}} vem crescido para lidar e mandar mais tráfego para mais Pods do backend, limitações da API original se tornaram mais visíveis.
```

Comment:
In this specific case, you don't need to translate `Service` because it's related to a Kubernetes resource instead of a "Serviço" in Portuguese.
```suggestion
A Endpoints API oferece uma maneira simples e direta de rastremento de endpoints de rede no Kubernetes. Infelizmente à medida que os clusters Kubernetes e os {{< glossary_tooltip text="Services" term_id="service" >}} vem crescendo para lidar e mandar mais tráfego para mais Pods do backend, limitações da API original se tornaram mais visíveis.
```